### DISCUSSION TITLE
audio and/or video files extensions support

### DISCUSSION THREAD

**Discussion by paulolafoudre** [sampotts/plyr#2092] [Q&A]
Hello,

I am wondering if there are audio and/or video files extensions not supported by the lib ?

And what's the best way to test it ?

Thanks

**Comment by sampotts** [comment]
Hey, this is probably a good place to start:
https://developer.mozilla.org/en-US/docs/Web/Media/Formats/Containers

**Reply by paulolafoudre** [reply]
Thanks ! I'll give it a try
